src/vmm/src/device_manager/persist.rs

```diff
@@ -360,16 +382,26 @@ impl<'a> Persist<'a> for MMIODeviceManager {
                                   slot: &MMIODeviceInfo,
                                   event_manager: &mut EventManager|
          -> Result<(), Self::Error> {
-            dev_manager
-                .slot_sanity_check(slot)
-                .map_err(Error::DeviceManager)?;
-
             let restore_args = MmioTransportConstructorArgs {
                 mem: mem.clone(),
                 device,
             };
             let mmio_transport =
                 MmioTransport::restore(restore_args, state).map_err(|()| Error::MmioTransport)?;
+
+            // At the moment we do not require exact re-allocation of id's via
+            // `dev_manager.irq_allocator.allocate_id()` and at the moment
```

Comment:
Maybe specify why: because we use the exact interrupt IDs, which are saved in the device's state.